spring-data-microservice-transaction-sample/README.md

```diff
@@ -465,84 +457,76 @@ if (updatedCreditTotal > customer.creditLimit) {
+`ScalarDbTwoPcRepository.executeTwoPcTransaction()` API, called on the Order Service, automatically performs preparations, validations and commits of both the local Order Service and the remote Customer Service. These steps are executed sequentially after the above CRUD operations successfully finish. The implementations to invoke `prepare`, `validate` and `commit` gRPC endpoints of the Customer Service need to be passed as parameters to the API. For reference, see [`OrderService.java`](order-service/src/main/java/sample/order/OrderService.java).
 
 ```java
-Collections.singletonList(
-  RemotePrepareCommitPhaseOperations.createSerializable(
-    this::callPrepareEndpoint,
-    this::callValidateEndpoint,
-    this::callCommitEndpoint,
-    this::callRollbackEndpoint
+TwoPcResult<PlaceOrderResponse> result = orderRepository.executeTwoPcTransaction(txId ->{
+    ...
+  },
+
+  Collections.singletonList(
+    RemotePrepareCommitPhaseOperations.createSerializable(
+      this::callPrepareEndpoint,
+      this::callValidateEndpoint,
+      this::callCommitEndpoint,
+      this::callRollbackEndpoint
+    )
   )
-)
+);
 ```
 
 ![Sequence Diagram of High Level 2PC API](images/seq-diagram-high-level-2pc-api.png)
 
-In the `prepare` endpoint of the Customer Service, it resumes and prepares the transaction (the code is [here](customer-service/src/main/java/sample/customer/CustomerService.java#L122-L126)):
+In the `prepare` endpoint of the Customer Service, it resumes and prepares the transaction with `ScalarDbTwoPcRepository.prepareTransactionOnParticipant()`. For reference, see [`CustomerService.java`](customer-service/src/main/java/sample/customer/CustomerService.java).
 
 ```java
-execTwoPcOperation(request.getTransactionId(), false, responseObserver, "Payment", () -> {
-  // Prepare the transaction
-  customerRepository.prepare();
-});
+customerRepository.prepareTransactionOnParticipant(request.getTransactionId());
 ```
 
-The transaction is resumed in `execTwoPcOperation()` as shown above.
-
-
-In the `validate` endpoint of the Customer Service, the microservice resumes and validates the transaction (the code is [here](customer-service/src/main/java/sample/customer/CustomerService.java#L131-L135)):
+In the `validate` endpoint of the Customer Service, it resumes and validates the transaction with `ScalarDbTwoPcRepository.validateTransactionOnParticipant()`. For reference, see [`CustomerService.java`](customer-service/src/main/java/sample/customer/CustomerService.java).
```

Comment:
Does `it` mean ``the `validate` endpoint``? If so, how about we change this sentence as follows?:

```suggestion
In the `validate` endpoint of the Customer Service, the endpoint resumes and validates the transaction by using `ScalarDbTwoPcRepository.validateTransactionOnParticipant()`. For reference, see [`CustomerService.java`](customer-service/src/main/java/sample/customer/CustomerService.java).
```